@here/harp-mapview/lib/poi/PoiManager.ts

```diff
@@ -136,75 +136,21 @@ export class PoiManager {
             );
 
             if (isLineMarkerTechnique(technique) && positions.count > 0) {
-                this.addLineMarker(tile, poiGeometry, technique, techniqueIndex, positions);
+                this.addLineMarker(
+                    tile,
+                    poiGeometry,
+                    technique as LineMarkerTechnique,
```

Comment:
Minor: the as LineMarkerTechnique is not needed, b/c typescipt can deduce the type from the condition above.